doc/src/manual/interfaces.md

```diff
@@ -391,6 +391,24 @@ something other than 1), you should specialize `indices`. You should also specia
 so that the `dims` argument (ordinarily a `Dims` size-tuple) can accept `AbstractUnitRange` objects,
 perhaps range-types `Ind` of your own design. For more information, see [Arrays with custom indices](@ref).
 
+## [Strided Arrays]
+
+| Methods to implement                            |                                        | Brief description                                                                     |
+|:----------------------------------------------- |:-------------------------------------- |:------------------------------------------------------------------------------------- |
+| `stride(A, i::Int)`                             |                                        | Return the distance in memory (in number of elements) between adjacent elements in dimension k.    |
+| `Base.unsafe_convert(::Type{Ptr{T}}, A)`        |                                        | Return the native address of an array.                                         |
+
+
+A strided array is a subtype of `AbstractArray` whose entries are stored in memory with fixed strides.
+Provided the element type of the array is compatible with BLAS, a strided array can utilize BLAS and LAPACK routines
+for more efficient linear algebra routines.
+
+A typical example of a user-defined strided array is one that wraps a standard `Array`
+with additional structure.
+
+
```

Comment:
I'd add a warning here too about how dangerous it is to implement these methods if the underlying storage is *not* actually strided. Otherwise the following might happen:
- user calls a function. Gets a MethodError due to missing implementation of `strides`
- "helpfully" implements `strides` to fix the method error

It might be useful to give some concrete examples of arrays and categorize them as to whether they have strided memory representation. Examples:
```julia
1:5   # not strided (there is no storage associated with this array, could segfault if implemented `strides`)
collect(1:5)  # is strided
A = [1 5; 2 6; 3 7; 4 8]  # is strided
V = view(A, 1:2, :)   # is strided
V = view(A, 1:2:4, :)   # is strided
V = view(A, [1,2,4], :)   # is not strided
```